docs/en/pipeline-mgmt/basic.md

```diff
@@ -0,0 +1,26 @@
+# Basic configuration
+
+For the clickstream project, you can specify the basic configuration of the data pipeline and set the following configurations:
+
+* **AWS Region**: select the region where the pipeline is located. If you specify the region where AWS services are not available, the feature will be disabled by default. Check [the region table][region-table] for feature availability.
+* **VPC**: specify the VPC where the compute resources of the pipeline will be located. The VPC needs to meet the below criteria for running the pipeline workload.
+    {%
+      include-markdown "./vpc-prerequisites.md"
+    %}
+* **Data collection SDK**: specify the SDK type that the client uses.
+    - If you choose **Clickstream SDK**, you can check the [SDK manual][clickstream-sdks] for available Clickstream SDKs.
+    - If you choose **Third-Party SDK**, the solution has built-in support for Google Tag Manager for server-side tagging. You can follow up on [Guidance for Using Google Tag Manager for Server-Side Website Analytics on AWS][gtm-guidance] to set up the GTM server-side servers on AWS. For other third-party SDKs, you need to follow up on [this step][custom-plugin] to use the custom transformer plug-in.
```

Comment:
If you choose **Third-Party SDK**,  you need to follow up on [this step][custom-plugin] to add a custom transformer plug-in to map the data to solution schema (if data modeling & reporting are needed). Note that the solution has built-in support for Google Tag Manager for server-side tagging, you can follow up on [Guidance for Using Google Tag Manager for Server-Side Website Analytics on AWS][gtm-guidance] to set up the GTM server-side servers on AWS.